### ISSUE TITLE
Project Overview

### ISSUE THREAD

**Issue by Elementator** [amosproj/amos2023ws04-pipeline-manager#1]
## User story
1. As a User
2. I want to upload my raw data
3. So that it get analyzed and store

## Acceptance criteria

- [ ] Can upload data
- [ ] Data gets analyzed by data pipeline
- [ ] Raw data and analyzed data are store in S3 bucket
- [ ] Database has the relation between data and metadata

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
